R/drive_cp.R

```diff
@@ -74,7 +74,11 @@ drive_cp <- function(file,
                      name = NULL,
                      ...,
                      overwrite = NA,
-                     verbose = TRUE) {
+                     verbose = deprecated()) {
+  if (lifecycle::is_present(verbose)) {
```

Comment:
This is the first of a gazillion such insertions, which is why I want `warn_for_verbose()` to figure out the function from which it was called.